library/build.gradle

```diff
@@ -17,6 +17,11 @@ android {
   }
 }
 
+// Disable Javadoc task as it is not compatible with Kotlin
+tasks.withType(Javadoc).all {
```

Comment:
Is there any way to make it work with kotlin? This seems a bit heavy-handed

Reply:
I could invest more time into it but for right now I think it is better if the project actually assembles. :-) Let me try one more thing...